components/dropdown/README.md

```diff
@@ -207,7 +207,7 @@ dropdown.addEventListener('click', function() {
       </d2l-menu-item>
       <d2l-menu-item text="Stars &amp; Galaxies"></d2l-menu-item>
       <d2l-menu-item text="The Night Sky"></d2l-menu-item>
-      <d2l-menu-item text="The Universe" id="last-item"></d2l-menu-item>
+      <d2l-menu-item text="The Universe"></d2l-menu-item>
     </d2l-menu>
   </d2l-dropdown-menu>
 </d2l-dropdown-button>
```

Comment:
Do we want to mention these new slots in the README?

Reply:
I know initially we didn't want to broadcast the capability but that seems to have changed with the use-cases that came up in discussion, so I can throw something in. 😃